### ISSUE TITLE
Disable suggestions for TS 'any' and 'unknown' type

### ISSUE THREAD

**Issue by ErkoKnoll** [microsoft/monaco-editor#1866]
Currently the default behaviour for TS 'any' and 'unknown' types is that suggestions won't be triggered after typing a dot, but when explicitly asked for (CTRL+Space) then random suggestions will be displayed that TS language service thinks could be used, most of the time these random suggestions are not applicable and therefore misleading.

Is there a way to disable random suggestions, and if not perhaps such a feature could be added?

**Comment by alexdima** [comment]
Is VS Code behaving in the same way?

**Comment by ErkoKnoll** [comment]
Yes, at least with default settings, maybe there are settings that can overwrite such a behaviour.
